src/components/common/section-templates/shared/title-description-cards-section/TitleDescriptionCardsSection.module.scss

```
@@ -119,10 +121,13 @@
 }
 
 .description-field {
+    width: 100%;
+
     textarea {
         @include type.body-1-regular;
         font-size: 16px;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)